xAPI.md

```diff
@@ -770,16 +770,42 @@ The table below lists all properties of the Verb Object.
 	</tr>
 </table>
 
-###### Requirements
+###### Verb Id Requirements
 
-* The display property MUST be used to illustrate the meaning which is already determined by the Verb IRI.
 * A system reading a Statement MUST use the Verb IRI to infer meaning.
+* The IRI contained in an id SHOULD contain a human-readable portion which SHOULD provide meaning enough 
+to disambiguate it from other similar(in syntax) Verbs.
+* A single Verb IRI MUST NOT be used to refer to multiple meanings.
+
+###### Verb Display AP Requirements
+* The display property SHOULD be used by all Statements.
+* The display property MUST be used to illustrate the meaning which is already determined by the Verb IRI.
+
+###### Verb Display LRS Requirements
+The requirements below relate to the display property as returned by the LRS via the API.  
+
+* When queried for statements with a format of "exact", the LRS MUST return the Display property 
```

Comment:
My thinking was that as verbs are not mentioned in the format: canonical description, the only possible interpretations are that (as I understood it) the LRS returns what's in the statement always, OR (as I think @fugu13 understood it) the LRS returns a different display depending on the format requested. Either way, with a format of 'exact' the display is exactly as included or omitted in the statement, so this can be a MUST. Any ambiguity only related to 'canonical' and 'ids' formats. I'm not too precious about MUST vs. SHOULD* though, so if you feel strongly I'll change it. 

I don't see the format as the same as serialisation and don't see these changes coming under that banner. Changing the content of language maps is a much bigger step than translating a timestamp in a different timezone. Translating a timestamp will always give the same result and can be reverted; that's not the case for canonical activity definitions or verb displays. 

These changes are based on the fact that the format query parameter doesn't mention verbs and that makes it ambiguous (as noted above, I thought it was clear that the LRS should always return exact verb display, but since @fugu13 read it otherwise I guess it's ambiguous). 

Your comments on statement immutability raise larger issues around the format parameter and statement immutability. I've raised an issue here: https://github.com/adlnet/xAPI-Spec/issues/573